lauren_vicknair/homework/calculator.rb

```diff
@@ -0,0 +1,35 @@
+puts "What do want from me?"
+answer = gets.chomp.downcase
+
+if ["add", "subtstract", "multiply", "divide", "remainder"].include?(answer)
```

Comment:
Great error handling! I would add one thing - try wrapping this into a loop so that your user can keep trying if they submit the wrong choice.

Check this out:

```
until ["add", "subtstract", "multiply", "divide", "remainder"].include?(answer)
  puts "I cannot do that"
  puts "Seriously, what do want from me?"
  answer = gets.chomp.downcase
end
```